### DISCUSSION TITLE
能否提供表格--明细行，在手机端等小屏状态下的展开功能

### DISCUSSION THREAD

**Discussion by taihangyishui** [dotnetcore/BootstrapBlazor#4915] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

能否让Table明细行，在手机等小屏状态下，增加向“选择”按钮一样的功能键，用于展开明细行。

### Describe the solution you'd like

同上。

### Additional context

![image](https://github.com/user-attachments/assets/abc849ab-01d8-4577-96fe-9e49a9d33932)

**Comment by bb-auto** [comment]
@taihangyishui Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@taihangyishui 假如提供了按钮，怎么展开呢？UI 如何设计？

**Comment by ArgoZhang** [comment]
@taihangyishui 需要一个思路来搞定 UI 布局，一直没有好办法

**Comment by taihangyishui** [comment]
![image](https://github.com/user-attachments/assets/734ec055-0630-4c4b-bad2-c8477f29012d)
![image](https://github.com/user-attachments/assets/314f9a05-3df1-4702-a441-ceac8e1ff8bb)
如上两个方案是否可行，第一张图是我发现原来功能，想使用链接完成的。我感觉第二张图，如果带有明细行(DetailRowTemplate)即自动增加展开按钮，并可向下展开。

**Comment by ArgoZhang** [comment]
@taihangyishui 以第二图为例，会丢失父子关系，滚屏后阅读难度增加，然后三层 四层 怎么办呢？这个方案我不太能接受

**Comment by densen2014** [comment]
小屏幕我倾向于用 Drawer/Dialog 弹窗显示明细表

```
<TableColumn>
 <Button Text="@Localizer["查看"]" />
</TableColumn>

<Drawer><Table>明细</Table></Drawer>
```
